docker_config/dockerfiles/Dockerfile.default-gpu

```diff
@@ -94,22 +78,13 @@ RUN python3 -m pip install -U \
       scikit-image \
       nltk
 
-RUN pip2 install spacy
-RUN python -m spacy download en
 RUN python3 -m spacy download en
 
-RUN pip2 install -U \
-      tensorflow-gpu==1.12.0 \
-      tensorboard \
-      keras
 RUN python3 -m pip install -U \
```

Comment:
For both dockerfiles: we should combine all pip install commands into a single command, and also use the `--no-cache-dir` option to save space. See https://stackoverflow.com/questions/45594707/what-is-pips-no-cache-dir-good-for (we can probably keep the pip install command at the end with apex as-is, though)